transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Service/TranscriptService.java

```diff
@@ -23,21 +24,33 @@ public class TranscriptService {
 
     private ApiRepository apiRepository;
 
-//    //Stores the APIKey in the apiAccess.properties files
-//    //Switched to static
-//    public static void storeAPIKey(String apiKey) {
-//    }
-//
-//
-//    //Stores the APIVersion in the apiAccess.properties files
-//    //Switched to static
-//    public static void storeAPIVersion(String apiVersion) {
-//
-//    }
+    public static ArrayList<String> getIntents() throws IOException {
+        ArrayList<String> list = new ArrayList<>();
+    //Intent 1
+        String intent_1 = getJSONContent().get(0).get(1).get(0).replaceAll("[:{\"}]","");
```

Comment:
Maybe try elaborate in the documentation on the reason for the indices used. Why for intent 1 do we need get(0).get(1).get(0) for example?